packages/@aws-cdk/aws-codepipeline-actions/lib/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './lambda/invoke-action';
 export * from './manual-approval-action';
 export * from './s3/deploy-action';
 export * from './s3/source-action';
+export * from './action'; // for some reason, JSII fails building the module without exporting this class
```

Comment:
I actually don't want to export this class - but if I don't, JSII fails building with an error. Any ideas how to sidestep that...?

Reply:
Corrected in the newest revision.

Reply:
My mistake - it is *not* in fact resolved. When I marked the class as `@internal`, the build for `@aws-codepipeline` succeeded, but then `@app-delivery` failed saying the CloudFormation Action don't implement the `IAction` interface. So I reverted the change, and I'm back to exporting the class.